.setup/bin/setup_sample_courses.py

```diff
@@ -489,7 +671,7 @@ def __init__(self, user):
         if 'rotating_section' in user:
             self.rotating_section = int(user['rotating_section'])
         if 'grading_registration_section' in user:
-            self.grading_registration_section = int(user['grading_registration_section'])
+            self.grading_registration_section = user['grading_registration_section']
```

Comment:
I added support for multiple grading registration sectiosn. By having the typecast we were limiting TAs to a max of 1 section per course, but this is too inflexible.